test/iteration.jl

```diff
@@ -112,7 +112,7 @@ end
     @test copy(erv[1]) == (y3=33, y1=53, y4=43)
     df.z = 1
     @test length(erd[1]) == 5 # the added column is reflected
-    deletecols!(df, [4,5])
+    select!(df, Not([4,5]))
```

Comment:
Can you add a basic test for `deletecols!` in test/deprecated.jl?

Reply:
added